addon/components/inputs/checkbox-array.js

```diff
@@ -0,0 +1,47 @@
+import AbstractInput from './abstract-input'
+import computed, {readOnly} from 'ember-computed-decorators'
+import layout from 'ember-frost-bunsen/templates/components/frost-bunsen-input-checkbox-array'
+
+export default AbstractInput.extend({
+  // == Component Properties ===================================================
+
+  classNames: [
+    'frost-bunsen-input-checkbox-array',
+    'frost-field'
+  ],
+
+  layout,
+
+  // == State Properties =======================================================
+
+  selected: [],
+
+  // == Computed Properties ====================================================
+
+  @readOnly
+  @computed('bunsenModel')
+  options (bunsenModel) {
+    return bunsenModel.items.enum.map((key) => {
```

Comment:
In the event the user omits `enum` in their model this will crash and burn. I'd make this a bit more robust with something like:

```js
import Ember from 'ember'
const {get} = Ember

…

  options (bunsenModel) {
    const items = get(bunsenModel, 'items.enum') || []

   return items.map((value) => {
      return {value}
    })
  },
```